### ISSUE TITLE
Brscan3 as well

### ISSUE THREAD

**Issue by NorfairKing** [NixOS/nixpkgs#84802]
Scanners are available using `brscan4` but I need `brscan3` for my own scanner.
I'm trying some things out, but I have no idea what I'm doing so it would be nice if I could get some help.

**Comment by NorfairKing** [comment]
I think I got most of the way there, but I got blocked by #84806 and there's another issue;
brscan3 puts/expects files in `/usr/local/Brother/sane` while brscan4 puts/expects files in `/etc/opt/brother/scanner/brscan4`. The former is not under `/etc`, so I can't just use `environment.etc` and I'm not sure what to do with that.

**Comment by NorfairKing** [comment]
It turns out that I don't actually need this. Should I close this issue then?

**Comment by flokli** [comment]
As it's not in nixpkgs yet, and you don't need it packaged anymore, it's probably fine closing it for now. 

If someone else sees the need for it, they can open a new issue or file a PR fixing it ;-)
